Tutorials/CNTK_209A_Evaluation_of_Pretrained_Super-resolution_Models.ipynb

```diff
@@ -0,0 +1,634 @@
+    "                for w in range(coef * y + offset, coef * y + output_dims - offset):\n",
+    "                    for col in range(0, 3):\n",
+    "                        result[h][w][col] = patch[h - coef * x][w - coef * y][col]\n",
+    "            \n",
+    "            #pad top\n",
+    "            if(x == 0):\n",
+    "                for h in range(offset):\n",
+    "                    for w in range(coef * y, coef * y + output_dims):\n",
+    "                        for col in range(0, 3):\n",
+    "                            result[h][w][col] = patch[h][w - coef * y][col]\n",
+    "            \n",
+    "            #pad left\n",
+    "            if(y == 0):\n",
+    "                for h in range(coef * x, coef * x + output_dims):\n",
+    "                    for w in range(offset):\n",
+    "                        for col in range(0, 3):\n",
+    "                            result[h][w][col] = patch[h - coef * x][w][col]\n",
+    "                  \n",
+    "            #pad bottom\n",
+    "            if(x == img.height - range_x):\n",
+    "                for h in range(coef * img.height - offset, coef * img.height):\n",
+    "                    for w in range(coef * y, coef * y + output_dims):\n",
+    "                        for col in range(0, 3):\n",
+    "                            result[h][w][col] = patch[h - coef * x][w - coef * y][col]\n",
+    "            \n",
+    "            #pad right                \n",
+    "            if(y == img.width - range_y):\n",
+    "                for h in range(coef * x, coef * x + output_dims):\n",
+    "                    for w in range(coef * img.width - offset, coef * img.width):\n",
+    "                        for col in range(0, 3):\n",
+    "                            result[h][w][col] = patch[h - coef * x][w - coef * y][col]\n",
+    "            \n",
+    "            #reached bottom of image\n",
+    "            if(x == img.height - range_x):\n",
+    "                break\n",
+    "            #next step by x, we must not go out of bounds\n",
+    "            x = min(x + step_x, img.height - range_x)\n",
+    "        \n",
+    "        #reached right edge of image\n",
+    "        if(y == img.width - range_x):\n",
+    "            break\n",
+    "        #next step by y, we must not go out of bounds\n",
+    "        y = min(y + step_y, img.width - range_x)\n",
+    "    \n",
+    "    #save result\n",
+    "    imsave(outfile, result.astype(np.uint8))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "deletable": true,
+    "editable": true
+   },
+   "source": [
+    "<p>Now we load our pretrained models.</p>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {
+    "collapsed": false,
+    "deletable": true,
+    "editable": true
+   },
+   "outputs": [],
+   "source": [
+    "#load models\n",
+    "VDSR_model = load_model(os.path.join(\"pretrained_models\", \"VDSR.model\"))\n",
```

Comment:
We are currently loading models from our VM (this will obviously not work for checking-in). How should loading of the models look like for the check-in? Where do we put the models so they can be loaded in an appropriate way?